src/main/java/az/ingress/ms_order/util/OrderUtil.java

```diff
@@ -0,0 +1,25 @@
+package az.ingress.ms_order.util;
+
+import az.ingress.ms_order.dao.entity.OrderEntity;
+import az.ingress.ms_order.dao.entity.OrderItemEntity;
+
+import java.math.BigDecimal;
+
+public class OrderUtil {
```

Comment:
dont use GodObject . separate to NotificationUtil and AmountUtil for example and you can make this classes singleton